### ISSUE TITLE
old commands for producing SI units are deprecated and should be removed

### ISSUE THREAD

**Issue by jensenr30** [milesdai/TAoE3Solutions#25]
@milesdai 

The commands below are deprecated and should be removed, right?
If you agree, I can carefully work through Chapter 1 to remove/convert all the old unit commands over to the appropriate \SI{}{} command (or \si{} depending on the circumstances).

```
\newcommand{\M}{\ensuremath{\text{M}}} % Mega
\renewcommand{\k}{\ensuremath{\text{k}}} % kilo
\newcommand{\m}{\ensuremath{\text{m}}} % milli
\renewcommand{\u}{\ensuremath{\mu}} % micro
\newcommand{\n}{\ensuremath{\text{n}}} % nano

\newcommand{\V}{\ensuremath{\text{V}}}
\newcommand{\Ohm}{\ensuremath{\Omega}}
\newcommand{\W}{\ensuremath{\text{W}}}
\newcommand{\A}{\ensuremath{\text{A}}}
\newcommand{\F}{\ensuremath{\text{F}}}
\renewcommand{\H}{\ensuremath{\text{H}}}
\newcommand{\Hz}{\ensuremath{\text{Hz}}}
```

**Comment by milesdai** [comment]
Good catch! Will fix

**Comment by milesdai** [comment]
Fixed in c2bda342744e00f62a67d60044a8d4f92bf8f7a6.
